src/app/shared/orderBy.ts

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
This is probably copied from somewhere.
Best to use a library or dependency if possible. 
If not, best to add a link in the comments or similar